index.md

```diff
@@ -1,10 +1,10 @@
 # Introduction
-Reaction Commerce is a modern, open source commerce platform for businesses. It is a lightweight, real-time reactive commerce solution with the goal of simplifying and aligning the user experience to the shop owner/admin experience.
+Reaction Commerce is a modern, open source platform for today's premier ecommerce experiences. It is a lightweight, real-time reactive commerce solution that redefines, simplifies, and aligns both the experiences of the user and the vendor. 
 
-Reaction is built using Javascript and the [Meteor](https://meteor.com) framework on the [Node](https://nodejs.org) runtime and the npm package ecosystem. It is both an open source solution and a managed Platform as a Service (PaaS) that is available via [reactioncommerce.com](https://reactioncommerce.com).
+Reaction is built using Javascript and the [Meteor](https://meteor.com) framework on the [Node](https://nodejs.org) runtime and the npm package ecosystem. It is both an open source solution and a managed Platform as a Service (PaaS), available via [reactioncommerce.com](https://reactioncommerce.com).
 
-We welcome [contributions](https://reactioncommerce.com/contributors), ideas, feedback, and questions via [GitHub Issues](https://github.com/reactioncommerce/reaction/issues?state=open). For general discussion and support, check out our [chat room](https://gitter.im/reactioncommerce/reaction).
+We welcome feedback, [contributions](https://reactioncommerce.com/contributors), and questions via [GitHub Issues](https://github.com/reactioncommerce/reaction/issues?state=open). For general discussion, viist our [Gitter Chatroom](https://gitter.im/reactioncommerce/reaction).
```

Comment:
I'd like to direct questions and feedback to forums.reactioncommerce.com, GitHub issues should be used, as much as possible, for bug reporting and development tasks.  

typo: `viist`.